src/payment/strategies/cybersource/cybersource-payment-strategy.spec.ts

```diff
@@ -368,6 +368,24 @@ describe('CyberSourcePaymentStrategy', () => {
                     expect(error).toBeInstanceOf(MissingDataError);
                 }
             });
+
+            it('use iin as BinNumber if is vaulted instrument', async () => {
+                jest.spyOn(paymentActionCreator, 'submitPayment')
+                    .mockReturnValueOnce(of(createErrorAction(PaymentActionType.SubmitPaymentFailed, requestError)));
+                jest.spyOn(cardinalClient, 'getThreeDSecureData').mockReturnValue(Promise.resolve('token'));
+
+                payload = merge({}, getOrderRequestBody(), {
+                    payment: {
+                        methodId: paymentMethodMock.id,
+                        gatewayId: paymentMethodMock.gateway,
+                        paymentData: getVaultedInstrument(),
+                    },
+                });
+
+                await strategy.execute(payload);
+
+                expect(cardinalClient.getThreeDSecureData).toHaveBeenCalled();
```

Comment:
How is this proof of the `iin` being used?

Reply:
The description of the test may not be accurate, the idea is the use of VaultedInstrument as PaymentData. I'll update the test name to reflect that...

Reply:
Method `getThreeDSecureData` is called only when the strategy successfully place the order, and in the scenario of a Vaulted Instrument being passed as paymentData, the assertion of `getThreeDSecureData` is the proper one